.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
+  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travis-coveralls/master/travis-coveralls.sh
+script:
+    - bash -ex .travis-opam.sh
+    - if [ ! -z ${XS_COV} ]; then bash -ex travis-coveralls.sh || true; fi
```

Comment:
If `XS_COV` is undefined, this leads to `[ ! -z ]` and there is no argument after the `-z`. This works but I would find it cleaner to use `[  ! -z "${XS_COV}" ]` such that an empty string would be the last argument in that case. The original code would break down if we added something like `[ ! -z ${XS_COV} -a ${XS_COV} gt 2 ]`.